### PR TITLE
global table routing

### PR DESCRIPTION
**Pull request by harshit-gangal**
Document the Vitess global table routing.

### CODE REVIEW

File: content/en/docs/reference/features/global-routing.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+title: Global Routing
+weight: 23
+---
+
+# Global Table Routing
+Vitess has an implicit feature of routing the queries to appropriate keyspace based on the table specified in the `from` list.
```
Comment (sougou) [inline content/en/docs/reference/features/global-routing.md]: to the appropriate keyspace

File: content/en/docs/reference/features/global-routing.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+This works only for unique table names provided in the [VSchema](https://vitess.io/docs/concepts/vschema/).
+
+This feature only works when no default keyspace is set on the connection.
+
```
Comment (sougou) [inline content/en/docs/reference/features/global-routing.md:11]: Combine: This works only for unique table names provided in the [VSchema](https://vitess.io/docs/concepts/vschema/), and only when no default keyspace is set on the connection.

File: content/en/docs/reference/features/global-routing.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+| Database |
++----------+
+| ks       |
+| customer |
+| commerce |
++----------+
+3 rows in set (0.00 sec)
+```
+
+`ks` and `customer` are sharded keyspace and `commerce` is unsharded keyspace.
```
Comment (sougou) [inline content/en/docs/reference/features/global-routing.md]: `ks` and `customer` are sharded keyspaces and `commerce` is an unsharded keyspace.

File: content/en/docs/reference/features/global-routing.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+| Tables_in_commerce |
++--------------------+
+| customer_seq       |
+| order_seq          |
+| product            |
++--------------------+
+3 rows in set (0.00 sec)
+```
+
+Without any default keyspace we can route to unique table like `corder`, `product` and `player` but cannot route to `customer`
```
Comment (sougou) [inline content/en/docs/reference/features/global-routing.md]: unique tables

File: content/en/docs/reference/features/global-routing.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+| player_id | bigint      | NO   | PRI | NULL    |       |
+| name      | varchar(50) | NO   |     | NULL    |       |
++-----------+-------------+------+-----+---------+-------+
+2 rows in set (0.00 sec)
+
+mysql> show columns from customer;
+ERROR 1105 (HY000): ambiguous table reference: customer
+```
+
+With default keyspace set to `customer` we can only query tables in `commerce` i.e `customer` and `corder`.
```
Comment (sougou) [inline content/en/docs/reference/features/global-routing.md]: the default keyspace

File: content/en/docs/reference/features/global-routing.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+| sku         | varbinary(128) | YES  |     | NULL    |       |
+| price       | bigint         | YES  |     | NULL    |       |
++-------------+----------------+------+-----+---------+-------+
+4 rows in set (0.00 sec)
+
+mysql> show columns from product;
+ERROR 1105 (HY000): table product not found
+```
+
+With default keyspace set, the queries can be routed to other keyspace by specifying the table qualifier.
```
Comment (sougou) [inline content/en/docs/reference/features/global-routing.md]: With a default keyspace set, the queries can be routed to other keyspaces by specifying the table qualifier.

**Review by sougou — COMMENTED** [review]
Some grammar suggestions.

It may be important to highlight how this differs from standard mysql. Maybe start off saying that mysql only allows you to use unqualified table names if a database is selected. But in vitess, this is not necessary because of the global routing feature.

File: content/en/docs/reference/features/global-routing.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+| Tables_in_commerce |
++--------------------+
+| customer_seq       |
+| order_seq          |
+| product            |
++--------------------+
+3 rows in set (0.00 sec)
+```
+
+Without any default keyspace we can route to unique tables like `corder`, `product` and `player` but cannot route to `customer`
```
Comment (deepthi) [inline content/en/docs/reference/features/global-routing.md]:
```suggestion
Without a default keyspace we can route to unique tables like `corder`, `product` and `player` but cannot route to `customer`
```

**Review by deepthi — COMMENTED** [review]
One more grammar suggestion.

**Review by deepthi — APPROVED** [review]
Looks like all feedback has been addressed. LGTM.

**Pull request merged**